clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -569,6 +569,35 @@ def YieldOp : CIR_Op<"yield", [ReturnLike, Terminator,
   ];
 }
 
+//===----------------------------------------------------------------------===//
+// BreakOp
+//===----------------------------------------------------------------------===//
+
+def BreakOp : CIR_Op<"break", [Terminator]> {
+  let summary = "C/C++ `break` statement equivalent";
+  let description = [{
+    The `cir.break` operation is used to cease the control flow to the parent
```

Comment:
Won't let me suggest an edit here for some reason, but the 'cease ...to the parent' text seems wrong.  DO we mean 'cede' control flow to the parent?